timemachine/fe/estimator.py

```diff
@@ -50,26 +76,92 @@ def equilibrate(integrator, barostat, potentials, coords, box, lamb, equil_steps
     return CoordsVelBox(coords=ctxt.get_x_t(), velocities=ctxt.get_v_t(), box=ctxt.get_box())
 
 
+def run_model_simulations(model, sys_params):
+    assert len(sys_params) == len(model.unbound_potentials)
+
+    bound_potentials = []
+    for params, unbound_pot in zip(sys_params, model.unbound_potentials):
+        bp = unbound_pot.bind(np.asarray(params))
+        bound_potentials.append(bp)
+
+    all_args = []
+    for lamb_idx, lamb in enumerate(model.lambda_schedule):
+
+        subsample_interval = 1000
```

Comment:
nit: can move this out of loop to quiet IDE's "might be referenced before assignment" warning at https://github.com/proteneer/timemachine/blob/20c83a8c5e6b334bdd50ba5100a41e1b1623fd6a/timemachine/fe/estimator.py#L124-L125

Reply:
Resolved and parameterized in [e19ead8](https://github.com/proteneer/timemachine/pull/700/commits/e19ead869f7eb48f47401759356432de3b5485c0)